examples/eq_runner_to_downstream/payload_v2/business/surveyresponse_0_0_3_with_cir.json

```diff
@@ -0,0 +1,145 @@
+{
+  "tx_id": "ea82c224-0f80-41cc-b877-8a7804b56c26",
+  "type": "uk.gov.ons.edc.eq:surveyresponse",
+  "version": "v2",
+  "data_version": "0.0.3",
+  "origin": "uk.gov.ons.edc.eq",
+  "flushed": false,
+  "submitted_at": "2016-05-21T16:37:56.551086",
+  "launch_language_code": "en",
+  "submission_language_code": "en",
+  "collection_exercise_sid": "9ced8dc9-f2f3-49f3-95af-2f3ca0b74ee3",
+  "cir_instrument_id": "c7b0cfa8-147d-4345-8948-813dcbc1539f",
+  "started_at": "2016-05-21T16:33:30.665144",
+  "case_id": "a386b2de-a615-42c8-a0f4-e274f9eb28ee",
+  "region_code": "GB-ENG",
+  "channel": "RAS",
+  "survey_metadata": {
+    "survey_id": "009",
+    "case_ref": "1000000000000001",
+    "case_type": "B",
+    "display_address": "ONS, Government Buildings, Cardiff Rd",
+    "employment_date": "2021-03-01",
+    "form_type": "0253",
+    "period_id": "202101",
+    "period_str": "January 2021",
+    "ref_p_end_date": "2021-06-01",
+    "ref_p_start_date": "2021-01-01",
+    "ru_name": "ACME T&T Limited",
+    "ru_ref": "49900000001A",
+    "trad_as": "ACME LTD.",
+    "user_id": "64389274239"
+  },
+  "data": {
+    "answers": [
+      {
+        "answer_id": "first-name",
+        "value": "John",
+        "list_item_id": "zGBdpb"
+      },
```

Comment:
Wonder if its worth trimming down some of the answers since the purpose of the example is the top level data? No strong view